doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -5,16 +5,82 @@
-* [1.2 Terminology.](#1.2)
-* [1.3 Approach](#1.3)
-* [1.4 Principles](#1.4)
-* [1.5 scope](#1.4)
-* [1.6 Vision](#1.4)
-* [1.7 Roadmap](#1.4)
+* [1.2 Terminology](#1.2)
+* [1.3 Principles](#1.3)
+  * [1.3.1 Cloud Native.](#1.3.1)
+* [1.4 Scope](#1.4)
+* [1.5 Vision](#1.5)
+* [1.6 Approach](#1.6)
+* [1.7 Roadmap](#1.7)
+
 
 <a name="1.1"></a>
 ## 1.1 Introduction
 
+> to be written
+
+
+<a name="1.2"></a>
+### 1.2 Terminology
+
+- **Scalable Applicatins**:
+- **Declarative APIs**:
+- **Platform**:
+
+
+<a name="1.3"></a>
+## 1.3 Principles
+
+Kubernetes Reference Architecture must obey to the following set of principles:
+- [CNTT Reference Model Principles](../../../ref_model/chapters/chapter01.md#1.3)
+- [CNTT Reference Architecture Principles](../../#principles)
+
+>Any Kubernetes specific principles needs to be added here.
+
+<a name="1.3.1"></a>
+### 1.3.1 Cloud Native Principles
+
+According to CNCF TOC (Technical Oversight Committee), following is the definition of Cloud Native:
+>CNCF Cloud Native Definition v1.0
+>Approved by TOC: 2018-06-11
+
+>“Cloud native technologies empower organizations to build and run **scalable** applications in modern, **dynamic environments** such as public, private, and hybrid clouds. Containers, **service meshes**, **microservices**, **immutable infrastructure**, and **declarative APIs** exemplify this approach.
+
+>These techniques enable **loosely coupled** systems that are **resilient**, **manageable**, and **observable**. Combined with **robust automation**, they allow engineers to make high-impact changes frequently and predictably with minimal toil.
+
+>The Cloud Native Computing Foundation seeks to drive adoption of this paradigm by fostering and sustaining an ecosystem of open source, vendor-neutral projects. We democratize state-of-the-art patterns to make these innovations accessible for everyone”
+
+The definition above is very applicatin centric and look at cloud native from the application point of view, in here, we will try and relate those definitions to the platforms of which cloud native applicatiosn run on top of.
+
+- **scalable**:
+- **dynamic environments**:
+- **service meshes**: 
+- **microservices**:
+- **immutable infrastructure**:
+- **declarative APIs**:
+- **loosely coupled**:
+- **resilient**:
+- **manageable**:
+- **observable**:
+- **robust automation**:
+
+<a name="1.4"></a>
+## 1.4 Scope
+
+> to be written
+<a name="1.5"></a>
+## 1.5 Vision
+
+<a name="1.6"></a>
+## 1.6 Approach
+
+
+<a name="1.7"></a>
+## 1.7 Roadmap
+
+
+## General thoughts
+
 This document (at this stage, not a RA, we're just using the template as a placeholder), discusses the role of Kubernetes in managing NFV software components (e.g. VNFCs). Application containers such as those managed by Kubernetes can be considered a Foundational PaaS capability<sup>1</sup> and so the following terms can all be thought of as being the topic of this document:
 - Kubernetes PaaS (Platform-as-a-Service)
```

Comment:
refer to next comment: definition of CaaS and PaaS is not consistent throughout the document
I propose starting the terminology chapter with these definition
Kubernetes: a tool to orchestrate containers
CaaS: Containers-as-a-Service layer providing container orchestration, and must have services to be able to manage CNFs at scale
practically it is K8s + minimal set of backends serving K8s abstract APIs
PaaS:  additional optional services on top of CaaS, providing value added centralized, and re-usable services for micro-service based CNFs requiring them
e.g. service mesh etc.

Reply:
I'm wondering, whether structure wise it would make sense to handle the RA for the PaaS separately compared to the CaaS. This would limit complexity in the work, and the PaaS would have to be run on top of a CaaS anyway (at least according to the definition above, i.e. when PaaS does not CONTAIN the CaaS.

Reply:
I'm fully onboard with that.
PaaS isn't necessarily part of the cloud infrastructure itself: it can be brought by an application, an independent vendor, multiple can co-exist on top of the same CaaS etc.

Reply:
I've removed PaaS from the terminology.